src/coreclr/src/jit/flowgraph.cpp

```diff
@@ -5983,6 +5989,29 @@ void Compiler::fgFindBasicBlocks()
         return;
     }
 
+    // If we are doing OSR, add an entry block that simply branches to the right IL offset.
+    // Might need to do something special if we're entering try regions?
+    if (opts.jitFlags->IsSet(JitFlags::JIT_FLAG_OSR))
```

Comment:
=> `opts.IsOSR()`

Reply:
(every usage of `JIT_FLAG_OSR` should go through that helper, right?)

Reply:
Yes, will update.